integration-tests/api-gateway/check/check-unhappy.test.ts

```diff
@@ -0,0 +1,87 @@
+import { checkEndpoint, createSession } from "../endpoints";
+import {
+  clearAttemptsTable,
+  clearItemsFromTables,
+} from "../../resources/dynamodb-helper";
+import { NINO } from "../env-variables";
+import { stackOutputs } from "../../resources/cloudformation-helper";
+
+jest.setTimeout(30000);
+
+describe("Given the session and NINO is invalid", () => {
+  let sessionId: string;
+  let personIDTableName: string;
+  let sessionTableName: string;
+  let output: Partial<{
+    CommonStackName: string;
+    StackName: string;
+    NinoUsersTable: string;
+    UserAttemptsTable: string;
+  }>;
+
+  afterEach(async () => {
+    output = await stackOutputs(process.env.STACK_NAME);
+    personIDTableName = `person-identity-${output.CommonStackName}`;
+    sessionTableName = `session-${output.CommonStackName}`;
+    await clearItemsFromTables(
+      {
+        tableName: personIDTableName,
+        items: { sessionId: sessionId },
+      },
+      {
+        tableName: `${output.NinoUsersTable}`,
+        items: { sessionId: sessionId },
+      },
+      {
+        tableName: sessionTableName,
+        items: { sessionId: sessionId },
+      }
+    );
+    await clearAttemptsTable(sessionId, `${output.UserAttemptsTable}`);
+  });
+
+  it("Should receive a 400 response when /check endpoint is called without session id value and optional header", async () => {
+    const session = await createSession();
+    const sessionData = await session.json();
+    sessionId = sessionData.session_id;
+    const check = await checkEndpoint({ "session-id": "" }, NINO);
```

Comment:
Same comment here, could we completely exclude the `"session-id"` rather than sending an empty string to make it a more accurate test?